website/docs/components/form/file-input/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,33 @@
+## Usage
+
+### When to use
+
+- Use a file input to let users quickly select files from their computer.
+
+### Types of File Inputs 
```

Comment:
I am not sure what this title means. What does "types" refer to, here? 🤔

Reply:
In that case, my suggestion below stands: let's move this whole section below, and call it "Browser differences" or something like this